services/spar/src/Spar/Sem/DefaultSsoCode/Cassandra.hs

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS_GHC -fplugin=Polysemy.Plugin #-}
+
+module Spar.Sem.DefaultSsoCode.Cassandra where
+
+import Cassandra
+import Imports
+import Polysemy
+import qualified Spar.Data as Data
+import Spar.Sem.DefaultSsoCode
+
+defaultSsoCodeToCassandra ::
+  forall m r a.
+  (MonadClient m, Member (Embed m) r) =>
+  Sem (DefaultSsoCode ': r) a ->
+  Sem r a
+defaultSsoCodeToCassandra =
+  interpret $
+    embed @m . \case
+      Get -> Data.getDefaultSsoCode
+      Store ip -> Data.storeDefaultSsoCode ip
+      Delete -> Data.deleteDefaultSsoCode
```

Comment:
I suppose at some point we can resolve `Spar.Data` into these intepreter modules, what do you think?

Reply:
That sounds good to me!